kubernetes/deployments_kubeclient.go

```diff
@@ -979,6 +984,27 @@ func (kc *kubeClient) getPods(namespace string, uid types.UID) ([]*v1.Pod, error
 	return appPods, nil
 }
 
+func (kc *kubeClient) getPodsQuota(pods []*v1.Pod) *app.PodsQuota {
+	cores := float64(0)
+	memory := float64(0)
+
+	for _, pod := range pods {
+		for _, container := range pod.Spec.Containers {
+			cpu, _ := quantityToFloat64(*container.Resources.Limits.Cpu())
+			mem, _ := quantityToFloat64(*container.Resources.Limits.Memory())
```

Comment:
Could we log and/or return any errors from quantityToFloat64?

Reply:
I've returned the errors now.